.github/cspell.json

```diff
@@ -3,43 +3,186 @@
   "language": "en",
   "flagWords": [],
   "words": [
+    "AHSL",
+    "ARGB",
     "ASDW",
+    "Akida",
+    "arial",
+    "artboard",
+    "aseprite",
+    "audioplayers",
+    "BGUG",
+    "ballons",
+    "bezier",
+    "bitfield",
+    "collidable",
+    "composability",
+    "coord",
+    "datas",
+    "deletable",
+    "draggable",
+    "draggables",
+    "Foo's",
+    "fullscreen",
+    "gamepads",
+    "hitboxes",
+    "hoverable",
+    "hoverables",
+    "ints",
+    "Kawabunga",
+    "kinect",
+    "klingsbo",
+    "LTRB",
+    "LTRBR",
+    "LTWH",
+    "Lukas",
+    "multitap",
+    "mutators",
+    "NPCs",
+    "Nakama",
+    "padracing",
+    "patreon",
+    "queryable",
+    "RGBO",
+    "raycasting",
+    "rects",
+    "respawned",
+    "rgba",
+    "SVGs",
+    "schnurber",
+    "skia",
+    "strikethrough",
+    "tappable",
+    "tappables",
+    "Tavian",
+    "tiled's",
+    "viewport's",
+    "WASD",
+    "WBMP",
+    "widget's",
+    "abelian",
+    "ambiguate",
+    "antialiasing",
+    "artboard",
+    "asprite",
+    "autofocus",
     "backtick",
     "backticks",
+    "ballons",
```

Comment:
not a word.
It's also already included up in the list. If you sort the lines alphabetically, any duplicate entries would be much easier to see. Note that most IDEs have built-in command for sorting lines, so there's no need to do it manually.